exonum/src/blockchain/schema.rs

```diff
@@ -146,30 +146,44 @@ where
     }
 
     /// Returns latest committed block.
+    ///
+    /// # Panics
+    /// - If the "genesis block" was not created.
```

Comment:
Nitpick: I propose to use the same formatting for doc-comments as in standard library. Example:
```
    /// Discription...
    ///
    /// # Panics
    ///
    /// Panics if ...
    ///
    /// # Examples
```
https://doc.rust-lang.org/src/alloc/vec.rs.html#448